src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -55,92 +62,114 @@ class Liberty implements Plugin<Project> {
         project.task('libertyRun', type: RunTask, dependsOn: 'libertyCreate') {
             description = "Runs a Websphere Liberty Profile server under the Gradle process."
             logging.level = LogLevel.INFO
-            server = currentServer
+            server = project.liberty.server
         }
 
         project.task('libertyStatus', type: StatusTask, dependsOn: 'libertyCreate') {
             description 'Checks if the Liberty server is running.'
             logging.level = LogLevel.INFO
-            server = currentServer
+            server = project.liberty.server
         }
 
         project.task('libertyCreate', type: CreateTask, dependsOn: 'installLiberty') {
             description 'Creates a WebSphere Liberty Profile server.'
-            outputs.file { new File(getUserDir(project), "servers/${project.liberty.serverName}/server.xml") }
+            outputs.file { new File(getUserDir(project), "servers/${project.liberty.server.name}/server.xml") }
```

Comment:
Won't this line be problematic for the old style liberty closure where the object gets set in the afterEvaluate method?  In that case will outputs.file get set to "servers/null/server.xml which will never exist?